src/extension/features/accounts/capture-native-search-shortcuts/settings.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: 'CaptureNativeSearchShortcuts',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Capture native search shortcuts',
+  description:
+    "Make pressing default browser search shortcuts (Ctrl+F/Cmd+F) focus in YNAB's account transaction input field. Restoring focus back to previously active element or field can be done by pressing Escape",
```

Comment:
I'm not comfortable overriding default browser shortcuts. Though I do like the idea of this feature.

Let's change the hotkey to something simple like literally just the letter `f` or `s`, if you add the listener to `document`, I don't think it'll trigger when users are editing a textbox you'll want to double check that.

This change would also get rid of the need to detect the platform which is a win in my book.